packages/react-searchbox/src/components/SearchBox.js

```diff
@@ -563,25 +580,41 @@ class SearchBox extends React.Component {
                           },
                           true
                         )
-                      : this.popularSuggestionsList.map((sugg, index) => (
-                          <li
-                            {...getItemProps({ item: sugg })}
-                            key={`${index + this.suggestionsList + 1}-${
-                              sugg.value
-                            }`}
-                            style={{
-                              backgroundColor: this.getBackgroundColor(
-                                highlightedIndex,
-                                index + this.suggestionsList
-                              )
-                            }}
-                          >
-                            <SuggestionItem
-                              currentValue={currentValue}
-                              suggestion={sugg}
-                            />
-                          </li>
-                        ))}
+                      : (this.popularSuggestionsList || []).map(
+                          (sugg, index) => (
+                            <li
+                              {...getItemProps({ item: sugg })}
+                              key={`${index + this.suggestionsList + 1}-${
```

Comment:
@anik-ghosh-au7 What is `this.suggestionsList`? If it is an array then use `this.suggestionsList.length` for keys everywhere.